lib/components/search_user/search_user.dart

```diff
@@ -1,4 +1,3 @@
-import 'package:flutter/cupertino.dart';
```

Comment:
idk who has been importing both Cupertino and Material

Reply:
oops